### PR TITLE
[FIX][pre][code] CSS problem

### PR DESCRIPTION
**Pull request by eaba**

### CODE REVIEW

**Comment by eaba** [comment]
I want to create another branch for `footer`? @Aaronontheweb

**Comment by Aaronontheweb** [comment]
> I want to create another branch for `footer`? @Aaronontheweb

that should be fine. Can you post screenshots of the new homepage here?

**Comment by eaba** [comment]
**before**:

image

 **after**:

image

**Comment by Aaronontheweb** [comment]
> I want to create another branch for `footer`? @Aaronontheweb

Based on our convo it in Slack, sounds like problems with the footer are the result of this PR. That needs to be fixed too before we can merge this.

**Comment by eaba** [comment]
https://getakka.net/api/Akka.Cluster.TestKit.html https://getakka.net/api/Akka.Actor.Internal.html

maybe just increase the height? Footer new PR?

**Comment by eaba** [comment]
image

File: docs/template/styles/docfx.js
Comment (Aaronontheweb) [inline docs/template/styles/docfx.js]: @eaba this was disabled so the footer would stay persistent at the bottom of the site. Why did you add it back?

**Review by Aaronontheweb — CHANGES_REQUESTED** [review]
Can you please explain why you made these changes?

**Review by Aaronontheweb — CHANGES_REQUESTED** [review]
Can you please explain why you made these changes?

**Comment by eaba** [comment]
I will revert it. First, I thought that the new changes are docfx v2.67.0. Second, @Arkatufus give me idea yesterday, I tried it, not the problem. Removing `footer` solved it!

**Comment by Aaronontheweb** [comment]
> I will revert it. First, I thought that the new changes are docfx v2.67.0. Second, @Arkatufus give me idea yesterday, I tried it, not the problem. Removing `footer` solved it!

thanks @eaba - I'll try running this again locally.

File: docs/docfx.json
Comment (Aaronontheweb) [inline docs/docfx.json:80]: Need to add all three of these deleted settings back

**Review by Aaronontheweb — CHANGES_REQUESTED** [review]
LGTM, provided that we add the required `docfx.json` settings back

**Review by Aaronontheweb — APPROVED** [review]

**Pull request merged**
